### ISSUE TITLE
NTR: Amniotic fluid leakage during gestation (Gestation & perinatal history)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7862]
New term request Amniotic fluid leakage during gestation (elthat_151204161844): In utero; pregnancy; maternal; mother; prenatal; prenatally  UMLS: 0269800

**Comment by pnrobinson** [comment]
(T. Sparks): PROM (prelabor rupture of membranes): PROM occurs at 37 weeks or later, and occurs prior to the onset of labor at term. PMID: 32080050.
PPROM (preterm premature rupture of membranes): PPROM occurs at less than 37 weeks. PPROM can happen spontaneously, during preterm labor, or following a procedure such as amniocentesis. PMID: 15167875.

I would defer to you about the parent term, but these do not always result in oligohydramnios or lower amniotic fluid since it can be regenerated quickly (ie in cases of a small leak).

**Comment by pnrobinson** [comment]
@MickeySegal
This term [Premature rupture of membranes HP:0001788](https://hpo.jax.org/app/browse/term/HP:0001788) is probably a synonym. If not, please describe the differential meaning.

**Comment by MickeySegal** [comment]
I agree with the approach of T. Sparks that this is PPROM, not PROM, and deserves a separate finding.  We use it for Amnion rupture sequence and Oligohydramnios sequence.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000310
- label: Preterm premature rupture of membranes
- parents: [Premature rupture of membranes (HP:0001788)](https://hpo.jax.org/app/browse/term/HP:0001788)
- pmid: PMID:15167875
- def: PPROM (preterm premature rupture of membranes) occurs at less than 37 weeks of gestation. PPROM can happen spontaneously, during preterm labor, or following a procedure such as amniocentesis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7862
- synonyms: PPROM(ABBREVIATION)
